### PR TITLE
Issue 432 fix

### PR DESCRIPTION
**Pull request by david-puglielli**

### CODE REVIEW

**Comment by david-puglielli** [comment]
So should we add use_strict to the other conversion functions as well (like convert_string_from_utf16)?

**Comment by yitam** [comment]
> So should we add use_strict to the other conversion functions as well (like convert_string_from_utf16)?

Do you think it's necessary at this point? Or something we can revisit as needed?

**Comment by david-puglielli** [comment]
I say revisit as needed, no need to add it yet.

File: test/functional/sqlsrv/sqlsrv_connStr.phpt [hunk 1]
```diff
@@ -77,89 +95,10 @@ $c = connect(array(
 if ($c !== false) {
     fatalError("sqlsrv_connect(4) should have failed");
 }
-print_r(sqlsrv_errors());
+checkErrors($utf16Error);
 
 echo "Test succeeded.\n";
 
 ?>
 --EXPECTF--
```
Comment (yitam) [inline test/functional/sqlsrv/sqlsrv_connStr.phpt]: Glad you finally fixed this test, but one minor point. Please change from EXPECTF to EXPECT

**Comment by david-puglielli** [comment]
Issue #432 fixed.

**Pull request merged**